migrate/migrate.go

```diff
@@ -184,6 +190,20 @@ func (m *Migrator) LoadMigrations(path string) error {
 		if err != nil {
 			return err
 		}
+		if m.options.RequireForwardMigration {
+			containsSQL := false
+			for _, v := range strings.Split(upSQL, "\n") {
+				// Only account for regular single line comment.
+				if len(strings.TrimSpace(v)) != 0 &&
+					!strings.HasPrefix(v, "--") {
```

Comment:
What if a comment also has spaces before it

Reply:
Fixed